### PR TITLE
[14.0][FIX] pos_*: avoid located CI issues

### PR DESCRIPTION
**Pull request by chienandalu**
Ensure test correctness when a different default localization is set on the CI as the existing pricelist could be in a different currency and that will cause all kind of test failures.

cc @Tecnativa TT46702

### CODE REVIEW

**Review by legalsylvain — COMMENTED** [review]
thanks ! 

still draft ?

**Comment by chienandalu** [comment]
Ready for review. I was testing them in batch in a local CI :)

File: pos_cash_move_reason/tests/test_pos_cash_move_reason.py [hunk 1]
```diff
@@ -13,8 +13,28 @@ def setUpClass(cls):
...
+                    )
+                ],
+            }
+        )
+        cls.config = cls.env.ref("point_of_sale.pos_config_main").copy(
+            {
+                "available_pricelist_ids": [(6, 0, cls.pricelist.ids)],
+                "pricelist_id": cls.pricelist.id,
+            }
+        )
```
Comment (legalsylvain) [inline pos_cash_move_reason/tests/test_pos_cash_move_reason.py:37]: why not use core function ? 

https://github.com/odoo/odoo/blob/14.0/addons/point_of_sale/tests/common.py#L596

**Review by legalsylvain — COMMENTED** [review]
naive question inline ?

File: pos_cash_move_reason/tests/test_pos_cash_move_reason.py (hunk 1 above)
Comment (chienandalu) [reply pos_cash_move_reason/tests/test_pos_cash_move_reason.py:37]: I've given it a shot but it looks that class pulls too many stuff to just use it out of the box without much refactor. As we're moving to future versions with these customer, I'd let the research for `16.0`

**Review by legalsylvain — APPROVED** [review]
OK !

/ocabot Merge patch

**Pull request merged**